metrotranscom/doorway
# fix: move stylesheet to correct path

# Pull Request Template

## Issue Overview

This PR addresses #issue


## Description

The import of the sign-in.module.scss was coming from the sites/public directory. This causes an issue when trying to build the partner site in docker because the public and partner sites are not pulled into each other.

## How Can This Be Tested/Reviewed?

The partner and public site should still properly build and have the form sign in page have correct styling

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
